integration-tests/cli/inventories-exporter.it.js

```diff
@@ -81,7 +81,8 @@ describe('StockExporter CLI', () => {
       )
       expect(stdout && stderr).toBeFalsy()
       const data = await fs.readFile(csvFilePath, { encoding: 'utf8' })
-      expect(data).toMatchSnapshot()
+      const expectedData = `12345,20,inventory-custom-type,integration tests!! arrgggh,${new Date().getFullYear()}`
```

Comment:
ooh... the pain behind that "arrgggh"

Reply:
lol yeah... its been there for ages though and I dont wanna change it in this PR